client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -51,24 +51,28 @@ function getHoldMessages( siteSlug, translate ) {
 			supportUrl: 'https://support.wordpress.com/domains/',
 		},
 		NO_WPCOM_NAMESERVERS: {
-			title: translate( 'No WordPress.com name servers' ),
-			description: translate( 'Use WordPress.com name servers on your custom domain to resolve.' ),
-			supportUrl: 'https://support.wordpress.com/domain-helper/',
+			title: translate( 'Domain not using WordPress.com name servers' ),
+			description: translate( 'Your domain must use WordPress.com name servers to support custom code. ' +
+			'Follow step 2 in "add domain mapping" to resolve this.' ),
```

Comment:
This could be more clear.

"Add domain mapping" (probably better to keep the same case as the support page) doesn't have any "internal" steps, but is itself the first step of the page.
You could be talking about:

- `Click the Upgrade button next to “Already own a domain?”`
which is the second line of the "Add domain mapping" section.

- `2. Ask your domain provider to update your DNS settings`
which is the second step of the entire page.

Reply:
I'll update this to link to the section as well